### ISSUE TITLE
world gen issues in jenkins build 2.2.2.25 through 2.2.2.297a09c (latest)

### ISSUE THREAD

**Issue by ate0ate** [TheCricket/Chisel-2#130]
It looks as though the recently added 1.8 blocks, other than diorite, are not generating in a new world. Also marble and limestone are missing from world gen as well. 
 Using forge version 10.13.2.1284

Edit: To clarify it seems that only one stone type generates per world now. Either andesite, diorite, or granite will generate in any world at any one time. Starting a new world usually seems to result in a different stone type generating, but only that one type.

**Comment by JadeCreeper** [comment]
I am using the same build and i can find all three of the new 1.8 stone types, except for limestone and marble.

**Comment by JadeCreeper** [comment]
The stone types seem to be biome-specific.

**Comment by ate0ate** [comment]
I'm not sure what's happening for me, but I definitely only have one stone type per world. I am using BOP and some other mods as well, but things were working quite well prior to the 1.8 world gen changes made in recent chisel builds. I cannot seem to find Limestone or Marble at all in any biome type.

**Comment by xeronut** [comment]
Bug occurs on Forge versions 1232 and 1272 as well, for what it's worth.  I surfed around a fresh world for almost an hour to test again on Forge 1272 + Biomes O Plenty.  Plains, Crag, Bog, Moor, Snowy Forest, Ocean, Scrubland, Vanilla Desert.. anywhere I went it was Limestone.  Started another world up, traveled around for about 5 minutes and quickly noticed any Chisel surface rocks were Granite instead.  Not sure what to make of it.

**Comment by tterrag1098** [comment]
Everything is generating in my test world. Sounds like an issue with BoP.

**Comment by ate0ate** [comment]
Okay, after further testing there's definitely seems to be something going on on chisels end. In my heavily modded world which is a BoP worldtype I only get one stone generating, diorite. This holds true in the deep dark dimension and the twilight forest. I stripped out everything and tested ONLY	Chisel2_DEV-2.2.2.d8882fb and forge 1286 in a default world type. Everything worked perfectly. However, when I tested just chisel and forge in an amplified world type, the only stone that generated was limestone. Seems like there's an issue with any other world type besides default, regardless of mods.

**Comment by ate0ate** [comment]
I just tried out Chisel2_DEV-2.2.2.6a11b3c.jar and it seems that now only one stone type will generate per world regardless of world type. I tried deleting my configs and then a completely fresh minecraft and forge 1286 install with the only mod being Chisel. No matter what world type is chosen only one stone type will generate per world. Which type of stone generates seems pretty random or maybe seed based since I've encountered nearly all of the various stones, but only one type per world. Either way, the generation issue isn't limited to world types other than default any longer in the latest jenkins build. Hope that helps some and I thank you for all the effort put into this amazing mod:)
